doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -85,12 +85,14 @@ The Requirements Exceptions lists the Reference Model (RM) requirements and/or R
 ## 1.4 Scope
 
 The scope of this particular Reference Architecture can be described as follows (the capabilities themselves will be listed and described in subsequent chapters), also shown in Figure 1-1:
+
 - Kubernetes capabilities required to conform to the Reference Model requirements
 - Support for CNFs that consist wholly of containers
 - Support for CNFs that consist partly of containers and partly of VMs, both of which will be orchestrated by Kubernetes
+- **Kubernetes Cluster lifecycle management**: including Cluster creation/upgrade/scaling/deletion, and node customisation due to workload requirements.
 
 The following items are considered **out of scope**:
-- **Kubernetes Cluster lifecycle management**: Since it is not considered to be "visible" to a CNF, it should not be included.
```

Comment:
I think Kubernetes LCM shoud _not_ be in the scope or RA2 as it does not affect the runtime charastheristics of the workload.

Reply:
This is about Cluster LCM which is critically important. The cluster scaling (part of cluster LCM) is important to workloads.

The objective here should be cloud agnostic cluster LCM.

Reference Architectures are not only about workloads but also about Operators ability to create and manage the environments for the workloads.

Reply:
> This is about Cluster LCM which is critically important. The cluster scaling (part of cluster LCM) is important to workloads.

I'm disagree. It's (very) important to operators, but I don't see the value to operators of defining a standard for this as the scaling criteria of a cluster is dependant on the non functional requirements of a particular team or organisation. 

Personally,  I would put this in the category of "in scope" as the security stuff, I.e. we can have some guidance but not part of the spec as it's a non functional behaviour not something that affects the "run time" interfaces that the workload consume,  or how those interfaces are managed.

Reply:
> Reference Architectures are not only about workloads but also about Operators ability to create and manage the environments for the workloads.

Some are, it depends on the scope. Not all Reference Architectures have the same scope.

Reply:
@pgoyal01 @rgs shall we add to the agenda and discuss tomorrow?

Reply:
yup it's there already https://wiki.anuket.io/display/HOME/2021-04-29+-+%5BAnuket+RA2%5D+-+Meeting+Agenda+and+Minutes

Reply:
added